bigtable/client/CMakeLists.txt

```diff
@@ -101,11 +101,15 @@ add_library(bigtable_client_testing
         testing/internal_table_test_fixture.h
         testing/internal_table_test_fixture.cc
         testing/mock_data_client.h
+        testing/inprocess_data_client.h
+        testing/inprocess_admin_client.h
         testing/mock_response_stream.h
         testing/table_integration_test.h
         testing/table_integration_test.cc
         testing/table_test_fixture.h
-        testing/table_test_fixture.cc)
+        testing/table_test_fixture.cc
+        testing/embedded_server_test_fixture.h
```

Comment:
In general we try to keep these lists in alphabetical order, please consider moving them, or removing them, see below.

Reply:
fixed